content/en/blog/2022/metrics-announcement.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I agree, but unfortunately this will impact the URL for the post on opentelemetry.io, which has already been direct linked by many of the vendors that are cross-posting this